README.md

```diff
@@ -196,6 +200,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [myQShor](https://github.com/Michaelvll/myQShor) - Quantum implementation of Shor's algorithm.
 
 **Julia**
+- [QAOA.jl](https://github.com/FZJ-PGI-12/QAOA.jl) - Implementation the Quantum Approximate Optimization Algorithm (QAOA) in Julia.
```

Comment:
The same package should not be duplicated (see first point of guidelines). If a package could belong to multiple categories, pick the best one.